wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights: abbreviate high totals

Fixes #n/a

To test:
- Go to Stats > Insights.
- Verify large numbers are abbreviated.
- Uses the same logic as Latest Post Summary https://github.com/wordpress-mobile/WordPress-iOS/pull/10426. 

NOTE: Follower totals are only available in string format, so they cannot be abbreviated yet. I've updated https://github.com/wordpress-mobile/WordPress-iOS/issues/10562 to reflect what's needed.

![abbrev](https://user-images.githubusercontent.com/1816888/49319581-b7d3b700-f4ba-11e8-9ffb-ea8e6a5fdff2.png)
